src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -647,7 +648,9 @@ public static function getAttributeData($product)
                 } else {
                     $attributeData['attribute_names'][$index] = $contentVar->get('label');
                 }
-                $attributeData['attribute_position'][$index] = count($attributeData['attribute_position']);
+
+                $weight = WooCommerceAttributeMetadata::getMetadata($attribute->id, 'weight', true, 'DESC') ?? 1;
```

Comment:
```suggestion
                $weight = WooCommerceAttributeMetadata::getMetadata($attribute->id, 'weight', true, 'DESC') ?? 0;
```

Maybe better to have 0 as default?

Reply:
Changed fallback to 0.